features/gateway_injection.md

```diff
@@ -0,0 +1,146 @@
+# Feature
+
+**Feature name:** Gateway Injection
+
+**Primary lead(s):** @howardjohn
+
+**Short description:** Injecting gateway deployments to enabled minimal configuration and simplified operations.
+
+---
+
+## Experimental
+
+### Requirements:
+
+- [ ] [User stories](insert_your_link_here) reviewed in a work group meeting.
+
+- [x] [RFC Authored] - [Injector V2](https://docs.google.com/document/d/1Rmp9B3DDypgMCau-YAqidx_r3qvKLZj6jUX-t22zj84/).
+
+- [x] [Documentation](https://preliminary.istio.io/latest/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental) and [here](https://preliminary.istio.io/latest/docs/setup/additional-setup/external-controlplane/) for enabling and using the feature.
```

Comment:
Couple comments:
1) First doc is about custom sidecar templates, it does not mention using this for gateways or provide an example. So I think it doesn't really qualify as documentation for this feature.
2) Second doc is about external control planes, is this only usable with external control planes? That doesn't sound right.

Reply:
https://github.com/istio/istio.io/pull/9706 is specifically for this, I will update this PR with a link to that for docs once its merged.

For 2, it is already the documented way to use gateways external control plane, but can be used in general. I think the above PR will clarify this a lot hopefully